### ISSUE TITLE
Always defaults

### ISSUE THREAD

**Issue by ekkis** [smhg/express-locale#4]
I have the following lines on a page request:

    console.log(req.header('Accept-Language'));
    console.log(req.locale);

which produce the following output:

> sv-SE,sv-FI;q=0.8,en-US;q=0.5,en;q=0.3
> { code: 'en_GB', source: 'default' }

I've configured the module like this:

    app.use(locale({
        priority: ['accept-language', 'cookie', 'domain', 'default']
    }));

so clearly I'm interested in the 'accept-language' first and my browser is providing it as `sv-SE` but the module returns the default.  what have I missed to configure properly?

**Comment by ekkis** [comment]
in looking more closely I see that the code is calling `req.acceptedLanguages` which returns undefined.  as I look at the req object I see no such field.  perhaps the code should be calling `req.header('Accept-Language')` and parsing the results? how is this supposed to work?

**Comment by ekkis** [comment]
I also notice that setting my language preference to Swedish (in Chrome) sets the language to 'sv' but includes no locale, so whereas I might have 'sv-SE' (Sweden), I could also have 'sv-FI' (Finland) but without locale I don't know how to properly format a price (because I don't know the currency).  that seems to be browser dependent though as Chrome doesn't set it but Firefox does.  so is there a way to tell the locale when the browser has not set it?

**Comment by smhg** [comment]
The `req.acceptedLanguages` issue is Express 4-specific and solved in c17d9013e0834cb1239509a71ad0371657202393. It hasn't been published to npm as we first need to add Express 3 and Express 4 specific tests.
Should be coming up shortly.
Can I ask you to put your second question in a separate issue?

**Comment by smhg** [comment]
Add this to your `package.json` to test the recent changes:
```
"express-locale": "git://github.com/smhg/express-locale#c17d901"
```
